code-challenges.js

```diff
@@ -57,12 +102,35 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
+const q4 = (array1, var1) => {
+    for (let i = 0; i < array1.length; i++) {
```

Comment:
This for loop looks beautiful, however, if you look at the code that follows it, you are never actually utilizing the for loop functionality.  The built-in method lastIndexOf essentially checks each element in the array removing the need to use a for loop.